sql_generators/glean_usage/templates/metrics_clients_last_seen.query.sql

```diff
@@ -39,6 +39,10 @@ FROM
   _previous
 FULL JOIN
   _current
-USING
-  (client_id,
-    sample_id)
+ON
+  _previous.client_id = _current.client_id AND
+  _previous.sample_id = _current.sample_id AND
+  (
+    _previous.normalized_channel = _current.normalized_channel OR 
+    (_previous.normalized_channel IS NULL AND _current.normalized_channel IS NULL) 
+  )
```

Comment:
@jklukas does this seem reasonable?

Reply:
This looks reasonable, though I'm not 100% sure without running it and validating the output. It's worth double-checking that we get the right result here before merging.